docs/configuration.asciidoc

```diff
@@ -212,6 +212,20 @@ You must use the query bar to filter for a specific environment in versions prio
 Enable or disable the agent. If set to false, then the Go agent does not send
 any data to the Elastic APM server, and instrumentation overhead is minimized.
 
+[float]
+[[config-recording]]
+=== `ELASTIC_APM_RECORDING`
+
+[options="header"]
+|============
+| Environment             | Default | Example
+| `ELASTIC_APM_RECORDING` | true    | `false`
+|============
+
+Enable or disable recording of events. If set to false, then the Go agent does
```

Comment:
```suggestion
Enable or disable recording of events. If set to false, then the Go agent does not
```

Reply:
Thanks, we are indeed - sorry for the waste of time.